testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -294,4 +325,19 @@ impl ClusterSwarm for ClusterSwarmKube {
         try_join_all(delete_futures).await?;
         Ok(())
     }
+
+    async fn get_grafana_baseurl(&self) -> Result<String> {
+        let data = Api::v1ConfigMap(self.client.clone())
+            .within(DEFAULT_NAMESPACE)
+            .get("workspace")
+            .await?
+            .data;
+        let workspace = data
+            .get("workspace")
+            .ok_or_else(|| format_err!("Failed to find workspace"))?;
+        Ok(format!(
+            "http://grafana.{}-k8s-testnet.aws.hlw3truzy4ls.com",
```

Comment:
Is there reason for having '-k8s-testnet' prefix here? Could we just keep same schema when domain is called "<workspace>. aws.hlw3truzy4ls.com"?

Reply:
Without the suffix, there would be a conflict when same tf workspace name is used to create a cluster using the old testnet and the new k8s testnet.

Reply:
If someone created same workspace with k8s and tf, that would be their mistake right?
I don't feel like we should carry over some confusing prefix just for this strange corner case

If we can just make URL look exactly same as it is right now, that would probably just simplify things further

Reply:
Since both https://github.com/calibra/libra-ops/tree/master/terraform/aws-k8s-testnet and https://github.com/calibra/libra-ops/tree/master/terraform/testnet create AWS resources in the same project, and they have separate terraform workspaces, it is easy to create terraform workspaces with the same name in both these terraform projects. For eg: I would create the `kush` tf workspace name in both the projects for my personal clusters. Once we completely move away from the old testnet, it makes sense to remove the suffix, but for now I think it is ok.